README.md

```diff
@@ -0,0 +1,37 @@
+#  Computer Science Society - Brand Resource Guidelines
+
+This document is intended to serve as a guide to the use of the CSS logos.
+
+## Colours
+
+|                   | Name      | Hex Code   | Red | Green | Blue | Use                                          |
+|-------------------|-----------|------------|-----|-------|------|----------------------------------------------|
+|![][col-dark-blue] | Dark Blue | `#2f3c63;` | 47  | 60    | 99   | Backgrounds                                  |
+|![][col-orange]    | Orange    | `#ff9c00;` | 255 | 156   | 0    | Logo Syntax Highlighting                     |    
+|![][col-light-blue]| Light Blue| `#58cad6;` | 88  | 202   | 214  | Logo Syntax Highlighting                     |            
+|![][col-white]     | White     | `#ffffff;` | 255 | 255   | 255  | Logo Syntax Highlighting                     |
+|![][col-green]     | Green     | `#03e421;` | 3   | 228   | 33   | Logo Syntax Highlighting                     |
+|![][col-yellow]    | Yellow    | `#fdff8a;` | 253 | 255   | 138  | Comment text when using Dark Blue Background |
+
+[col-dark-blue]: https://placehold.it/15/2f3c63/000000?text=+
+[col-orange]: https://placehold.it/15/ff9c00/000000?text=+
+[col-light-blue]: https://placehold.it/15/58cad6/000000?text=+
+[col-white]: https://placehold.it/15/ffffff/000000?text=+
+[col-green]: https://placehold.it/15/03e421/000000?text=+
+[col-yellow]: https://placehold.it/15/fdff8a/000000?text=+
+
+## Logos
+
+### Guild of Students and University of Birmingham Logo
+
+Follow the advice issued by the Guild Of Students on use of the Guild and University logos. 
+The advice is available through the Committee Admin page, on the Guild website.
+
+### CSS
+
+Where possible, the full logo should be used, and using the bold version of the logo where possible. They are available in this repository.
```

Comment:
Suggestion: "In all cases, the full logo should be used. The bold variant should be preferred, where possible, but is not required."